app/code/Magento/Catalog/Model/Product/Option/Value.php

```diff
@@ -225,16 +254,17 @@ public function saveValues()
     }
 
     /**
-     * Return price. If $flag is true and price is percent
-     *  return converted percent to price
+     * Return price.
+     *
+     * If $flag is true and price is percent return converted percent to price
      *
```

Comment:
I take this comment as an example of very good and less good use of it at the same time. 

When you say something like "if $flag is true..." it is very valuable but at the same time saying "Return price" on a method named `getPrice()` doesn't add any additional value.

Next time, I would recommend avoiding comments that repeat the name of the method, just use a comment to explain something that's not very explicit, that way you will add valuable info.

Reply:
@aleron75 thank you for the review! Unfortunately, code sniffers are forcing us to add some description to method annotation, so sometimes we have no other way then add some absolutely useless comment to old methods.